code/Qt/TrainWindow.cpp

```diff
@@ -54,19 +54,26 @@ TrainWindow::TrainWindow(int16_t* dataPtr){
     trainGroup->setLayout(trainLayout);
 
 
+    netGroup = new QGroupBox(tr("Neural Network"));
 
-    homeButton = new QPushButton("Save");
-    connect(homeButton, &QPushButton::clicked, [this](){this->closeWindow();});
+    nettrainButton = new QPushButton("Run Network");
+    connect(nettrainButton, &QPushButton::clicked, [this](){this->startNeuralNet();});
```

Comment:
do we want to close the window after training? have a button so we can safely close the window

Reply:
done